### PR TITLE
A literal shitposter PR

### PR DESCRIPTION
**Pull request by BobJoga**
## About The Pull Request

photocopiers are now able to print posters
if there is less than 10 toner it will fail to copy the poster and do an error poster instead

also added a bunch of new posters

## Why It's Good For The Game

poster spam time
image

## Changelog
:cl:
add: Photocopiers are now able to copy posters.
add: Added some new dumb posters.
/:cl:

### CODE REVIEW

File: code/game/objects/effects/contraband.dm [hunk 1]
```diff
@@ -181,6 +182,21 @@
...
+
+/obj/structure/sign/poster/gmod
+	name = "missing texture poster"
+	desc = "Looks like you'll have to install CS Source."
+	icon_state = "gmod"
+	never_random = TRUE
+	poster_item_name = "error poster"
+	poster_item_desc = "<span style='color:#d75050;'>Runtime in contraband.dm, line 183: Cannot read null.desc.</span>"
+	poster_item_icon_state = "rolled_poster"
+
```
Comment (jjpark-kb) [inline code/game/objects/effects/contraband.dm:199]: make this modular

File: code/game/objects/effects/contraband.dm [hunk 2]
```diff
@@ -946,4 +962,74 @@
...
+/obj/structure/sign/poster/official/rip_n_tear_borgs
+	name = "Rip and Tear"
+	desc = "A motivational poster, depicting a dogborg's head on a pike with a simple scrawled message RIP AND TEAR."
+	icon_state = "poster133"
+
+/obj/structure/sign/poster/official/cryptominer
+	name = "Cryptominer"
+	desc = "A poster, depicting a glowing red cryptomining supercomputer."
+	icon_state = "poster134"
+
```
Comment (jjpark-kb) [inline code/game/objects/effects/contraband.dm:1034]: make this modular

File: code/game/objects/effects/contraband.dm (hunk 1 above)
Comment (BobJoga) [reply code/game/objects/effects/contraband.dm:199]: will do

File: code/game/objects/effects/contraband.dm (hunk 2 above)
Comment (BobJoga) [reply code/game/objects/effects/contraband.dm:1034]: will do

**Pull request merged**